ImprovedAnvil/resources/dlg/cespy.d

```diff
@@ -2125,6 +2129,36 @@ IF ~~ THEN BEGIN treefist
       EXIT
   END
 
+  IF ~~ THEN BEGIN hexxat1
+    SAY @791 IF ~~ THEN REPLY @2 GOTO 13
+    IF ~PartyHasItem("ohhcloak") PartyGoldGT(24999) 
+	PartyHasItem("clck01") PartyHasItem("s!clck10")~
+      THEN REPLY @3
+      DO ~SetGlobal("Iahexxat01forged","GLOBAL",1)
+          TakePartyGold(25000)      DestroyGold(25000)
+          TakePartyItemNum("ohhcloak",1)   DestroyItem("ohhcloak")
+          TakePartyItemNum("clck01",1)   DestroyItem("clck01")
+          TakePartyItemNum("s!clck10",1)   DestroyItem("s!clck10")
+          GiveItemCreate("s!clck08",Player1,1,1,1)
+          CreateVisualEffect("spcrtwpn",[401.348])~
+      EXIT
+  END
+  
+  IF ~~ THEN BEGIN hexxat2
+    SAY @793 IF ~~ THEN REPLY @2 GOTO 13
+    IF ~PartyHasItem("s!clck08") PartyGoldGT(49999) 
+	PartyHasItem("clck01") PartyHasItem("dwdust") PartyHasItem("s!misc06") ~
```

Comment:
Maybe use a stronger cloak of protection in the second recipe? Feels too easy for such a powerful item